src/tasks/SwiftProcess.ts

```diff
@@ -136,6 +137,11 @@ export class SwiftPtyProcess implements SwiftProcess {
     }
 
     setDimensions(dimensions: vscode.TerminalDimensions): void {
+        // https://github.com/swiftlang/vscode-swift/issues/1074
+        // Causing weird truncation issues
+        if (process.platform === "win32") {
+            return;
+        }
         this.spawnedProcess?.resize(dimensions.columns, dimensions.rows);
```

Comment:
Does this cause the terminal output to be incorrectly sized in then Terminal tab of VS Code on mac/linux?

Reply:
Should only affect windows

Reply:
setDimensions is only called on Windows?